script/validate-categories/validate.rb

```diff
@@ -0,0 +1,37 @@
+require 'json'
```

Comment:
Why not use validate-data script itself?

Reply:
The reasons I plan on keeping the script separate
1. Modularity : The validate-data script already does multiple checks. We expect to add more checks in the validate-categories script, which will make validate-data bulky if combined.
2. Validate-Data is a mandatory check on PR, failing which will block PR merge. We intend Validate-categories to nudge the PR author with information on best practices for the values of the category field. There is no plan on failing the check.

Reply:
Ah! I think I get it now. category validation is an optional thing and we only put a PR message. Seems like we discussed it long back.

Reply:
@aparna-ravindra actually just re-thinking about it, currently we are only validating the template type category and not tech stack category. And if so, then we should fail the validation just like validate-data. And hence we can make this part of validate-data itself. Thoughts?